<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="InfoLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Narrow</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>База настроек не создана!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreateSDB">
        <property name="text">
         <string>Создать базу настроек</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreateAdmin">
        <property name="text">
         <string>Создать Администратора</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabMainWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1MW">
       <attribute name="title">
        <string>Аналитика</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutMainTab">
          <item>
           <widget class="QTextEdit" name="textEditListFullText"/>
          </item>
          <item>
           <widget class="QTabWidget" name="tabVLMTWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab1VLMT">
             <attribute name="title">
              <string>Регистратор</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="labelNumOfExe">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContentsCheckExe">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>449</width>
                   <height>133</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab2VLMT">
             <attribute name="title">
              <string>Информация</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTextEdit" name="textEditInfoReg"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutVLMT">
            <item>
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
            <item>
             <widget class="QCalendarWidget" name="calendarWidget"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2MW">
       <attribute name="title">
        <string>СПрУТ</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="textBrowserSPrUT"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_ImportFromExcel"/>
    <addaction name="action_ExportToExcel"/>
    <addaction name="action_Print"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_DB">
    <property name="title">
     <string>База данных</string>
    </property>
    <widget class="QMenu" name="menu_Connections">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Соединения</string>
     </property>
     <widget class="QMenu" name="menuSQLite">
      <property name="title">
       <string>SQLite</string>
      </property>
      <addaction name="action_NewSQLiteFile"/>
      <addaction name="action_SelectSQLiteFile"/>
     </widget>
     <addaction name="menuSQLite"/>
    </widget>
    <widget class="QMenu" name="menu_Connect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Соединиться</string>
     </property>
     <addaction name="action_ConnectSelSQLite"/>
    </widget>
    <addaction name="menu_Connections"/>
    <addaction name="menu_Connect"/>
    <addaction name="action_BackUp"/>
    <addaction name="action_Restore"/>
   </widget>
   <widget class="QMenu" name="menu_Tree">
    <property name="title">
     <string>Дерево</string>
    </property>
    <addaction name="action_AddRoot"/>
    <addaction name="action_DelRoot"/>
    <addaction name="action_AddBranche"/>
    <addaction name="action_DelBranche"/>
    <addaction name="action_AddItem"/>
    <addaction name="action_DelItem"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователи</string>
    </property>
    <widget class="QMenu" name="menu_Admin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Администратор</string>
     </property>
     <addaction name="action_AddAdmin"/>
     <addaction name="action_AdminMode"/>
    </widget>
    <addaction name="menu_Admin"/>
    <addaction name="action_NetworkRegistration"/>
    <addaction name="action_NetworkLogin"/>
    <addaction name="action_LocalRegistration"/>
    <addaction name="action_LocalLogin"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_DB"/>
   <addaction name="menu_Tree"/>
   <addaction name="menu_Help"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_ImportFromExcel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Импорт из Excel</string>
   </property>
  </action>
  <action name="action_ExportToExcel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Экспорт в Excel</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_BackUp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Резервная копия</string>
   </property>
  </action>
  <action name="action_Restore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Восстановление</string>
   </property>
  </action>
  <action name="action_AddRoot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить корень</string>
   </property>
  </action>
  <action name="action_AddBranche">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить ветку</string>
   </property>
  </action>
  <action name="action_DelBranche">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить ветку</string>
   </property>
  </action>
  <action name="action_DelRoot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить корень</string>
   </property>
  </action>
  <action name="action_AddItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить запись</string>
   </property>
  </action>
  <action name="action_DelItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </action>
  <action name="action_NetworkRegistration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сетевая регистрация</string>
   </property>
  </action>
  <action name="action_NetworkLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сетевой вход</string>
   </property>
  </action>
  <action name="action_LocalRegistration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Локальная регистрация</string>
   </property>
  </action>
  <action name="action_LocalLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Локальный вход</string>
   </property>
  </action>
  <action name="action_NewSQLiteFile">
   <property name="text">
    <string>Создать файл</string>
   </property>
  </action>
  <action name="action_SelectSQLiteFile">
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </action>
  <action name="action_ConnectSelSQLite">
   <property name="text">
    <string>С выбранным файлом SQLite</string>
   </property>
  </action>
  <action name="action_AddAdmin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Создать администратора</string>
   </property>
  </action>
  <action name="action_AdminMode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Режим &quot;Администратор&quot;</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
